src/https_client.c

```diff
@@ -63,6 +85,11 @@ static void https_fetch_ctx_init(https_client_t *client,
   if (logging_debug_enabled()) {
     curl_easy_setopt(ctx->curl, CURLOPT_VERBOSE, 1L);
   }
+#if defined(IP_TOS)
+  if (client->opt->dscp)
```

Comment:
Is the indentation wrong here or the parans? This is why I like parens around conditionals. :)

Reply:
Whooops!  Too much Python.